keps/sig-cli/2206-openapi-features-in-kustomize/kep.yaml

```diff
@@ -9,16 +9,11 @@ reviewers:
   - "@monopole"
   - "@pwittrock"
 approvers:
-  - n/a
+  - "@monopole"
+  - "@pwittrock"
 creation-date: 2020-12-21
 last-updated: 2020-01-15
-status: implementable
-see-also:
-  - n/a
-replaces:
-  - n/a
-superseded-by:
-  - n/a
+status: implemented
```

Comment:
Same comment about KEP status, applies to any that are not stable.

Reply:
TBH I find these values kinda tough to determine for Kustomize.  Kustomize the CLI is stable (v4), but its main API, Kustomization, is beta, and almost all features are part of that API. To further complicate things, the API module for using Kustomize in other programs has its own version and is alpha. None of these versions are actually tied to the k/k release cycle... but periodically (very infrequently in the past, but hopefully that's changing) the version of kustomize in kubectl gets bumped. So... it's complicated, and doesn't line up super nicely with the lifecycle Enhancements was designed for. 😅 

This particular feature is a field in Kustomization. It has been implemented and documented, and works without any feature gates. I'm sure it will evolve in small ways like any piece of code, but there's no major work remaining afaik. Therefore I don't see much value in holding the KEP open.

Reply:
agree with @KnVerey, the work we intended to do with this KEP is done so I think it can be closed